src/components/amount/Amount.tsx

```diff
@@ -421,7 +434,12 @@ const Amount: React.FC<AmountProps> = ({
   };
 
   useEffect(() => {
-    initRef.current();
+    try {
+      initRef.current();
+    } catch (err: any) {
+      logger.error('Error trying to init Amount view.');
+      logger.debug(err);
```

Comment:
Maybe this could be improved using:
```
const errStr = err instanceof Error ? err.message : JSON.stringify(err);
logger.error('[Amount] could not initialize view: ', errStr);
```

Reply:
Done. Thanks!